test/Tester/MembershipTests/LivenessTests.cs

```diff
@@ -275,12 +279,37 @@ public override TestCluster CreateTestCluster()
 
     public class LivenessTests_DynamoDB : LivenessTestsBase
     {
+        private static Lazy<bool> canConnectDynamoDb = new Lazy<bool>(() =>
```

Comment:
>canConnectDynamoDb [](start = 34, length = 18)

shouldn't this just use AWSTestConstants.CanConnectDynamoDb ? Or is there a difference between connection strings that I'm missing and is important?